
<launch>
    <!-- Include another launch file with arguments -->
    <include file="$(find test_interface)/launch/single_drone_rotation_mid360.launch">
    </include>

    <rosparam command="load" file="$(find optimalcontroller)/config/simulation.yaml" />
    <node pkg="optimalcontroller" type="constantSpeedControl" name="constantSpeedControl" output="screen" />
    
</launch>



